src/main/kotlin/com/theworkshopvlc/chupapuntes/categories/persistence/ICategoriesDAO.kt

```diff
@@ -0,0 +1,11 @@
+package com.theworkshopvlc.chupapuntes.categories.persistence
+
+import com.theworkshopvlc.chupapuntes.categories.domain.Category
+import org.springframework.data.jpa.repository.Query
+import org.springframework.data.repository.CrudRepository
+
+interface ICategoriesDAO : CrudRepository<Category, Long> {
+
+  @Query("SELECT c FROM Category c WHERE c.searchTitle LIKE '%' || :textTitle || '%'")
```

Comment:
We are using a full text search here with regex. Although this is really bad for performance we can keep it for the moment as our database won't be really big.